themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md

```diff
@@ -111,12 +163,28 @@ Remember how we can use the same program to define multiple stacks? Let's set
 the ports for this stack, which the Pulumi command line knows already from when
 you first initialized the project (it's the `dev` stack by default):
 
+{{< chooser language "typescript,python" / >}}
```

Comment:
Does this actually need a chooser?

Reply:
It does, but only because we want the user to be able to choose the right project template for their language, and this is the first appearance of a language chooser in the pathway.